eks/cluster-loader/configs/cluster-autoscaler/config.yaml

```diff
@@ -0,0 +1,131 @@
+# Constants
+# Currently set to 1 namespace.
+{{$NAMESPACES := DefaultParam .NAMESPACES 1}}
+{{$PODS_PER_NODE := DefaultParam .PODS_PER_NODE 10}}
+{{$QPS := DefaultParam .QPS 100}}
+{{$POD_GROUP_LABEL := DefaultParam .POD_GROUP_LABEL "cluster-loader"}}
+# Adjust the following two for your cluster -
+# kubectl get node -o json | jq -r ".items[].status.allocatable.cpu"
+{{$NODE_FREE_CPU := DefaultParam .NODE_FREE_CPU 1}}
+# kubectl get node -o json | jq -r ".items[].status.allocatable.memory"
+# In MB
+{{$NODE_FREE_MEM := DefaultParam .NODE_FREE_MEM 3840}}
+
+# Variables
+{{$NODES_PER_NAMESPACE := DivideInt .Nodes $NAMESPACES}}
+{{$PODS_PER_NAMESPACE := MultiplyInt $PODS_PER_NODE $NODES_PER_NAMESPACE}}
+{{$TOTAL_PODS := MultiplyInt $PODS_PER_NODE (MultiplyInt $NODES_PER_NAMESPACE $NAMESPACES)}}
+
+
+# Pod Resource proportional to node to ensure consistent triggering of scale-up
+{{$POD_CPU_MILLI := DivideFloat $NODE_FREE_CPU (MultiplyFloat $PODS_PER_NODE .9)}}
+{{$POD_MEMORY_MB := DivideFloat $NODE_FREE_MEM (MultiplyFloat $PODS_PER_NODE .9)}}
+
+name: simple-bring-up
+namespace:
+  number: 1
+tuningSets:
+- name: Uniform-QPS
+  qpsLoad:
+    qps: {{$QPS}}
+
+steps:
+- name: start-measurements
+  measurements:
+  - Identifier: ScaleUpTimer
+    Method: Timer
+    Params:
+      action: start
+      label: scaleUpTime
+  - Identifier: PodStartupLatency
+    Method: PodStartupLatency
+    Params:
+      action: start
+      labelSelector: group = {{$POD_GROUP_LABEL}}
+      threshold: 1h
```

Comment:
I don't think we need to fail the test on this.

Reply:
Got it